guides/common/modules/proc_creating-a-content-view.adoc

```diff
@@ -10,22 +10,18 @@ While you can stipulate whether you want to resolve any package dependencies on
 For more information, see xref:Resolving_Package_Dependencies_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create Content View*.
 . In the *Name* field, enter a name for the view.
 {Project} automatically completes the *Label* field from the name you enter.
 . In the *Description* field, enter a description of the view.
-. Optional: if you want to solve dependencies automatically every time you publish this Content View, select the *Solve Dependencies* check box.
+. In the *Type* field, select a *Component Content View* or a *Composite Content View*.
```

Comment:
@maximiliankolb In my Foreman instance, I don't believe it was cherry picked because this is what I get:
![Screenshot from 2022-05-24 15-55-10](https://user-images.githubusercontent.com/91690569/170121160-fb63d042-7db0-4a56-8126-34240e94afe7.png)
I went ahead and updated Component Content View to Content View anyway.

Reply:
Yes, I have created a PR for that hoping it gets merged and accepted. As I understand it right now, this would also require a new Katello 4.3.Z release.